rspace/src/main/scala/coop/rchain/rspace/history/Store.scala

```diff
@@ -40,7 +47,9 @@ object StoreInstances {
                 .setMaxReaders(config.maxReaders)
                 .open(config.path.toFile, config.flags: _*)
             }
-      dbi   <- Sync[F].delay { env.openDbi("db", MDB_CREATE) }
+      dbi <- Sync[F].delay {
```

Comment:
This change is unnecessary

Reply:
I don't recall to make this change. Wondering if the Scalafmt did this?